assets/src/app/videos/workflow/workflow_dialog.component.html

```diff
@@ -35,6 +35,14 @@ <h1 mat-dialog-title>Workflow configuration</h1>
                 [(ngModel)]=parameter.value
                 (change)=toNumber(parameter)>
             </mat-form-field>
+            <div *ngIf="parameter.type == 'boolean'" class="parameter" [ngClass]="{disabled: !step.enable}">
+              {{parameter.id | parameterLabel}}:
+              <mat-checkbox class="small"
+                color="accent"
+                [disabled]=!step.enable
+                [(ngModel)]=parameter.value
+                ></mat-checkbox>
```

Comment:
put text `{{parameter.id | parameterLabel}}:` inside to enable clic on label too.

Reply:
Done in 842d5fb!